src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -61,6 +62,23 @@ describe('Test suite for dictionary concepts components', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render the Concept Table without breaking', () => {
+    const props = {
+      concepts: [{ ...concepts, retired: true }],
+      loading: false,
+      org: {},
+      locationPath: {},
+      showDeleteModal: jest.fn(),
+      handleDelete: jest.fn(),
+      conceptLimit: 1,
+      closeDeleteModal: jest.fn(),
+      handleDeleteMapping: jest.fn(),
+      showDeleteMappingModal: jest.fn(),
+    };
+    const wrapper = mount(<ConceptTable {...props} />);
```

Comment:
I thought the test would expect something about the different display between retired and unretired concepts.

Reply:
The test checks to see if this component will render when retired concepts are present.
`concepts: [{ ...concepts, retired: true }],`

Reply:
If concepts are still present, but not retired, will this test fail?

Reply:
The test would pass. Let me make it more specific.

Reply:
So it is not testing what it is supposed to.

Reply:
Thanks for pointing it out. I have made it more specific. The test would now fail if there are no retired concepts. It looks for the exact HTML tag that creates strikethrough text.

Reply:
Shouldn't this correspondingly change? 'should render the Concept Table without breaking'

Reply:
Yes. I've changed it to "should contain strikethrough text for retired concepts"